examples/FecBikeTrainerDisplay/fe-c_native.ino

```diff
@@ -0,0 +1,209 @@
+/***********************************
+ * AntPlus Native Drive Fe-c Display
+ *
+ * Finds a nearby Indoor bike trainer
+ * And display content sent by it
+ *
+ * Author Charles-Antoine FOURNEL
+ ************************************/
+#include <Arduino.h>
+#include "ANT.h"
+#include "ANTPLUS.h"
+#include "bluefruit.h"
+#define BAUD_RATE 115200
+#define CHANNEL_0 0
+#define TOTAL_CHANNELS 1
+#define ENCRYPTED_CHANNELS 0
+
+const uint8_t NETWORK_KEY[] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08}; // get this from thisisant.com
```

Comment:
In one of your commits you have this number, you can squash the two commits so the history is removed or I can do it.